src/main/java/com/artipie/SliceFromConfig.java

```diff
@@ -299,6 +301,12 @@ static Slice build(
             case "docker-proxy":
                 slice = new DockerProxy(SliceFromConfig.HTTP, standalone, cfg, permissions, auth);
                 break;
+            case "deb":
+                slice = trimIfNotStandalone(
+                    settings, standalone,
+                    new LoggingSlice(new DebianSlice(cfg.storage(), permissions, auth, cfg.name()))
```

Comment:
@olenagerasimova I don't think we should wrap `DebianSlice` with `LoggingSlice` here, logging is enabled & configured on other level